locust/stats.py

```diff
@@ -84,7 +84,7 @@ class StatsErrorDict(StatsBaseDict):
 
 class StatsHolder(Protocol):
     name: str
-    method: str
+    method: str | None
```

Comment:
Why did this become optional?

Reply:
The `Total` / aggregated statistics is created with method set to `None`.
https://github.com/locustio/locust/blob/2ec2e70b18e98eb99b118b70e4229bfa04057710/locust/stats.py#L213
It was needed to solve a type issue for some of the sorting logic.

However, I see now that it should then probably be changed in `StatsEntry` as well, then,
https://github.com/locustio/locust/blob/2ec2e70b18e98eb99b118b70e4229bfa04057710/locust/stats.py#L291